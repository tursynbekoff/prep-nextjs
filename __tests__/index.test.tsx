import { render, screen } from '@testing-library/react'
import Home from '../pages/index'
import '@testing-library/jest-dom'

describe('Home', () => {
  it('renders a heading', () => {
    render(<Home />)

    const heading = screen.getByRole('heading', {
      level: 1,
      name: /Some text for test/i,
    })

    expect(heading).toBeInTheDocument()
  })
})