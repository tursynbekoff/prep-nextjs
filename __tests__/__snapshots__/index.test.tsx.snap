// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders homepage unchanged 1`] = `
<div>
  <main>
    Some text for test
  </main>
</div>
`;
